libkbfs/folder_branch_ops.go

```diff
@@ -2316,56 +2309,6 @@ func (fbo *folderBranchOps) createEntryLocked(
 		newBlock = &FileBlock{}
 	}
 
-	// Passthrough journal writes temporarily.
-	if excl == WithExcl {
-		if jServer, err := GetJournalServer(fbo.config); err == nil {
-			// Repeatedly flush and try to disable the journal. Since
-			// we hold the write lock, this shouldn't take more than
-			// one attempt very often (but could happen since block
-			// archives and removals don't take the write lock).
-			// TODO: this opens us up to timeout issues; perhaps
-			// investigate how we can re-order file system operations
-			// easily without compromising semantics too much (i.e.,
-			// to jump this operation to the front of the journal).
-			wasEnabled := false
-			for i := 0; i < 20; i++ {
-				err = WaitForTLFJournal(ctx, fbo.config, fbo.id(), fbo.log)
-				if err != nil {
-					return nil, DirEntry{}, err
-				}
-				wasEnabled, err = jServer.Disable(ctx, fbo.id())
```

Comment:
I'd like to leave it in, unless it complicates something in the future.  Seems like the right thing for Disable to do.